core/src/main/scala/org/apache/spark/scheduler/SparkListenerBus.scala

```diff
@@ -71,7 +71,6 @@ private[spark] trait SparkListenerBus
         listener.onNodeUnblacklisted(nodeUnblacklisted)
       case blockUpdated: SparkListenerBlockUpdated =>
         listener.onBlockUpdated(blockUpdated)
-      case logStart: SparkListenerLogStart => // ignore event log metadata
```

Comment:
Will this change the behavior of `SparkListenerBus`? Previously we ignored this event, but here with this change we will post this event through `listener.onOtherEvent` and user will get this event. But from the code this event is not accessible outside of Spark.

```
private[spark] case class SparkListenerLogStart(sparkVersion: String) extends SparkListenerEvent
```